bin/validateSchema.ts

```diff
@@ -0,0 +1,358 @@
+
+  // Validate against example files
+  ['./examples/get_visits_200_limit_1.json', './examples/get_visits_200_limit_500.json'].forEach((examplePath) =>
+    validateJson({
+      json: JSON.parse(fs.readFileSync(examplePath).toString()),
+      jsonName: examplePath,
+      validator: visitsResponseValidator,
+      schemaName: 'VisitsResponse',
+    })
+  );
+
+  // Validate against live Server API responses
+  for (const subscription of testSubscriptions) {
+    const client = new FingerprintJsServerApiClient({
+      apiKey: subscription.serverApiKey,
+      region: REGION_MAP[subscription.region || 'us'],
+    });
+
+    try {
+      const visitsResponse = await client.getVisitorHistory(subscription.visitorId);
+      validateJson({
+        json: visitsResponse,
+        jsonName: `🌐 Live Server API Visits Response for '${subscription.name}' > '${subscription.visitorId}'`,
+        validator: visitsResponseValidator,
+        schemaName: 'VisitsResponse',
+      });
+    } catch (error) {
+      console.error(error);
+      fail(`❌ Error while validating live Server API VisitsResponse for ${subscription.name}`);
+    }
+  }
+}
+
+/**
+ * Validates Webhook schema
+ */
+async function validateWebhookSchema() {
+  console.log('\nValidating Webhook schema: \n');
+  const webhookSchema = convertOpenApiToJsonSchema(OPEN_API_SCHEMA, '#/definitions/WebhookVisit');
+  const webhookValidator = ajv.compile(webhookSchema);
+
+  // Validate against example file
+  ['./examples/webhook.json'].forEach((examplePath) =>
+    validateJson({
+      json: JSON.parse(fs.readFileSync(examplePath).toString()),
+      jsonName: examplePath,
+      validator: webhookValidator,
+      schemaName: 'WebhookSchema',
+    })
+  );
+}
+
+/**
+ * Validates EventError403 schema
+ */
+async function validateEventError403Schema(testSubscriptions: TestSubscription[]) {
+  console.log('\nValidating EventError403 schema: \n');
+  const eventError403Schema = convertOpenApiToJsonSchema(OPEN_API_SCHEMA, '#/definitions/ErrorEvent403Response');
+  const eventError403Validator = ajv.compile(eventError403Schema);
+
+  // Validate against example file
+  ['./examples/get_event_403_error.json'].forEach((examplePath) =>
+    validateJson({
+      json: JSON.parse(fs.readFileSync(examplePath).toString()),
+      jsonName: examplePath,
+      validator: eventError403Validator,
+      schemaName: 'EventError403Schema',
+    })
+  );
+
+  // Validate against live Server API responses
+  for (const subscription of testSubscriptions) {
+    const client = new FingerprintJsServerApiClient({
+      apiKey: 'Wrong Server API Key',
+      region: REGION_MAP[subscription.region || 'us'],
+    });
+
+    try {
+      const eventResponse = await client.getEvent(subscription.requestId);
+      fail(`❌ Request for event ${eventResponse} in ${subscription.name} should have failed`);
+    } catch (error) {
+      // Node SDK adds "status" to the error response, just get rid of it and validate the rest
```

Comment:
Could we just fix Node SDK?

Reply:
Fix how? It's [this line causing the trouble ](https://github.com/fingerprintjs/fingerprintjs-pro-server-api-node-sdk/blob/4b5a30667c1e845796051210358a605b6658ae22/src/serverApiClient.ts#L64) but I think removing it would be a breaking change. 

We could maybe disable that behavior with a config option like "passthroughErrorsUnchanged" but maybe that's too hacky, wdyt?

Reply:
It sounds a bit hacky, yeah 😁. It looks okay for me, I think we can keep it as it is, maybe let's just add a comment to the code. WDYT @ilfa?

Reply:
We are going to make a major release anyway, let's remove this `status` field.
Error already contains a code.

We can keep this hack here before a major release.

Reply:
Thank you!